### PR TITLE
Fix registration page styling

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

File: app/controllers/user/registrations_controller.rb [hunk 1]
```diff
@@ -13,9 +12,9 @@ class User::RegistrationsController < Devise::RegistrationsController
   # end
 
   # GET /resource/edit
-  # def edit
-  #   super
-  # end
+  def edit
+    render 'edit', layout: 'user_admin'
```
Comment (allenwq) [inline app/controllers/user/registrations_controller.rb]: Do we need super ?

Comment (lowjoel) [inline app/controllers/user/registrations_controller.rb]: Super will cause a DoubleRenderError.

**Comment by allenwq** [comment]
@lowjoel One more issue, we need this layout for both edit and update, so that when update fails user are still in the same layout.

**Comment by lowjoel** [comment]
Nice one. Might not be so simple though.

**Pull request merged**